packages/dev/core/src/XR/features/WebXRHandTracking.ts

```diff
@@ -362,11 +367,15 @@ export class WebXRHand implements IDisposable {
 
         // Initialize the joint transform quaternions and link the transforms to the bones.
         for (let jointIdx = 0; jointIdx < this._jointTransforms.length; jointIdx++) {
-            const jointTransform = (this._jointTransforms[jointIdx] = new TransformNode(handJointReferenceArray[jointIdx], this._scene));
-            jointTransform.rotationQuaternion = new Quaternion();
+            this._jointTransforms[jointIdx] = this._jointTransforms[jointIdx] || new TransformNode(handJointReferenceArray[jointIdx], this._scene);

```

Comment:
How could `this._joinTransforms[jointIdx]` not be undefined here? Isn't this where these resources are first initialized?

Reply:
```suggestion
            this._jointTransforms[jointIdx] = new TransformNode(handJointReferenceArray[jointIdx], this._scene);
```

Reply:
It can't... fixed :-)